guides/source/configuring.md

```diff
@@ -641,7 +641,6 @@ See [Custom Configuration](#custom-configuration)
 
 Enables YJIT as of Ruby 3.3, to bring sizeable performance improvements. If you are
 deploying to a memory constrained environment you may want to set this to `false`.
-Additionally, you can pass a hash to configure YJIT options such as `{ stats: true }`.
```

Comment:
mhmm, not sure what happened here, will rebase (also `railties/lib/rails/application/finisher.rb`)